tsconfig.json

```diff
@@ -46,7 +46,8 @@
     // "rootDirs": [],                        /* List of root folders whose combined content represents the structure of the project at runtime. */
     "typeRoots": [
       "node_modules/@types",
-      "types"
+      "types",
+      "typings"
```

Comment:
What is the difference here? Why 2 separate local folders?

Reply:
See the comment above.

Reply:
Sure, my proposal would be to get rid of our custom one if it works using only e.g. types.

Reply:
"--if it works" seems to be the limiting factor. Starting to look like we need to figure out another way of including the file in the build since it still doesn't work as intended when the src folder isn't present.